recipes/paho-mqtt-c/all/patches/0003-fix-cmake-install.patch

```diff
@@ -1,221 +0,0 @@
- ADD_LIBRARY(common_obj OBJECT ${common_src})
--SET_PROPERTY(TARGET common_obj PROPERTY POSITION_INDEPENDENT_CODE ON)
--
--ADD_EXECUTABLE(MQTTVersion MQTTVersion.c)
--
--ADD_LIBRARY(paho-mqtt3c SHARED $<TARGET_OBJECTS:common_obj> MQTTClient.c)
--ADD_LIBRARY(paho-mqtt3a SHARED $<TARGET_OBJECTS:common_obj> MQTTAsync.c)
--
--TARGET_LINK_LIBRARIES(paho-mqtt3c ${LIBS_SYSTEM})
--TARGET_LINK_LIBRARIES(paho-mqtt3a ${LIBS_SYSTEM})
--
--TARGET_LINK_LIBRARIES(MQTTVersion paho-mqtt3a paho-mqtt3c ${LIBS_SYSTEM})
--SET_TARGET_PROPERTIES(
--    paho-mqtt3c paho-mqtt3a PROPERTIES
--    VERSION ${CLIENT_VERSION}
--    SOVERSION ${PAHO_VERSION_MAJOR})
--
--INSTALL(TARGETS paho-mqtt3c paho-mqtt3a
--    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
--    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
--    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
--INSTALL(TARGETS MQTTVersion
--    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
--
--IF (PAHO_BUILD_STATIC)
--    ADD_LIBRARY(paho-mqtt3c-static STATIC $<TARGET_OBJECTS:common_obj> MQTTClient.c)
--    ADD_LIBRARY(paho-mqtt3a-static STATIC $<TARGET_OBJECTS:common_obj> MQTTAsync.c)
--
--    TARGET_LINK_LIBRARIES(paho-mqtt3c-static ${LIBS_SYSTEM})
--    TARGET_LINK_LIBRARIES(paho-mqtt3a-static ${LIBS_SYSTEM})
--
--    INSTALL(TARGETS paho-mqtt3c-static paho-mqtt3a-static
--        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
--ENDIF()
- 
- INSTALL(FILES MQTTAsync.h MQTTClient.h MQTTClientPersistence.h MQTTProperties.h MQTTReasonCodes.h MQTTSubscribeOpts.h
-     DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
- 
- IF (PAHO_WITH_SSL)
-     SET(OPENSSL_SEARCH_PATH "" CACHE PATH "Directory containing OpenSSL libraries and includes")
--
--    IF (${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")
--      SET(OPENSSL_SEARCH_PATH "/usr/local/opt/openssl")
--    ENDIF (${CMAKE_SYSTEM_NAME} STREQUAL "Darwin")
--
--    IF (WIN32)
--      SET(OPENSSL_SEARCH_PATH "C:/OpenSSL-Win64")
--      SET(OPENSSL_SEARCH_LIB_PATH "${OPENSSL_SEARCH_PATH}/lib64")
--      IF (DEFINED ENV{PAHO_WINDOWS_BUILD_BIT})
--        IF ($ENV{PAHO_WINDOWS_BUILD_BIT} STREQUAL "x86")
--          SET(OPENSSL_SEARCH_PATH "C:/OpenSSL-Win32")
--          SET(OPENSSL_SEARCH_LIB_PATH "${OPENSSL_SEARCH_PATH}/lib32")
--        ENDIF ()
--      ENDIF ()
--    ELSE()
--      SET(OPENSSL_SEARCH_LIB_PATH "${OPENSSL_SEARCH_PATH}/lib64")
--    ENDIF ()
--
--    FIND_PATH(OPENSSL_INCLUDE_DIR openssl/ssl.h
--        HINTS ${OPENSSL_SEARCH_PATH}/include)
--    FIND_LIBRARY(OPENSSL_LIB NAMES ssl libssl ssleay32
--        HINTS ${OPENSSL_SEARCH_PATH}/lib ${OPENSSL_SEARCH_LIB_PATH})
--    FIND_LIBRARY(OPENSSLCRYPTO_LIB NAMES crypto libcrypto libeay32
--      	HINTS ${OPENSSL_SEARCH_PATH}/lib ${OPENSSL_SEARCH_LIB_PATH})
-+    SET(OPENSSL_ROOT_DIR "" CACHE PATH "Directory containing OpenSSL libraries and includes")
-+        
-+    find_package(OpenSSL REQUIRED)
-+    
-+    SET(OPENSSL_LIB ${OPENSSL_SSL_LIBRARY})
-+    SET(OPENSSLCRYPTO_LIB ${OPENSSL_CRYPTO_LIBRARY})
```

Comment:
https://github.com/eclipse/paho.mqtt.c/blob/v1.3.1/src/CMakeLists.txt#L115-L116

I need to revive this part of the patch for windows blah!